tests/unit/cmd/sca/test_scan.py

```diff
@@ -342,32 +340,32 @@ def test_sca_text_handler_ordering(patch_scan_all, cli_fs_runner):
 
 >>> : Incident 1 (SCA): bar@2.5.6
 Severity: Critical
-Summary: 
+Summary:
```

Comment:
If you remove the space, the test will fail. It should be ok with the space with the `noqa` at the end of the string